engine/src/main/scala/cromwell/engine/workflow/lifecycle/materialization/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -256,20 +267,20 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
     }
 
     val localFilesystemResolvers =
-      if (importLocalFilesystem) DirectoryResolver.localFilesystemResolvers(None)
+      if (importLocalFilesystem) DirectoryResolver.localFilesystemResolvers(baseWdl = None, NoopResolvedImportsStore)
```

Comment:
Would it make sense to have `NoopResolvedImportsStore` be the default value for `localFilesystemResolvers` so that `MaterializeWorkflowDescriptorActor.scala` doesn't have to know about it?

While it's correct to set it here, it is potentially distracting IMO